docs/setting-up/server/virtual-appliance.md

```diff
@@ -27,8 +27,8 @@ Most steps can be done with either a user interface or on the command line, but
 			- Disk 1 (sda): VMDK (SCSI, 40 GB)
 			- Disk 2 (sdb): VMDK (SCSI, 400 GB)
 	- Username/default password:
-		- `root`/`percona`
-    	- `admin`/`admin`
+		- `root`/`percona` (SSH)
+    	- `admin`/`admin` (PMM server account )
```

Comment:
Thanks. But admin/admin is also a CentOS Linux user account, and the user can connect with any account's SSH so long as they set up their keys. This clarification box is for the VM as a Linux instance.

Reply:
FYI The PMM User admin/admin is mentioned in section 5.